lib/src/ovis_util/util.c

```diff
@@ -90,6 +90,27 @@ static const char *get_env_var(const char *src, size_t start_off, size_t end_off
 	return name;
 }
 
+static const char *get_tilde_var(const char *src, size_t start_off, size_t end_off)
+{
+	static char name[100];
```

Comment:
This will make `str_repl_env_vars()` not thread-safe. I think we don't have any places that would access the path concurrently. However, we could make this thread-safe, e.g., make the function receiving the `char * name` and its size.

Reply:
get_env_var is also thread un-safe in the same way.
Whether we take this pull request or not, the use of the static name variable needs to be converted to a caller-supplied buffer.

Can fix both misuses with an additional commit on this pr if we decide to take this pr.